src/main/java/com/example/bookstore/service/BookService.java

```diff
@@ -10,4 +10,8 @@ public interface BookService {
     List<BookDto> getAll();
 
     BookDto getBookById(Long id);
+
+    void deleteById(Long id);
+
+    void updateById(Long id, CreateBookRequestDto createBookRequestDto);
```

Comment:
Return updated book.
We will have only one update method, so we can just update.
We can simplify variable name here.

```suggestion
    BookDto update(Long id, CreateBookRequestDto bookDto);
```